src/components/ProjectDetailsPanel/project-details-panel.module.css

```diff
@@ -27,11 +27,40 @@
     font-size: 25px;
     font-weight: 100;
 }
+.projectDetailsIcon{
+  width: 30px;
+  height: 30px;
+  margin-right: 8px;
+}
+.infoItems {
+  float:left;
+  display:inline;
+  width: 48%;
+  color: #989aa1;
+  margin-bottom: 5px;
+  margin-right: 2%;
 
+}
+.abstractTitle{
+  clear: both;
+  color: white;
+}
 .projectDetailsAbstract{
 
 }
 
+.projectDetailsLink{
+  float:left;
+  display:inline;
+  width: 48%;
+  margin-bottom: 5px;
+  margin-right: 2%;
+}
+
 .abstractText{
     color: white;
-}
\ No newline at end of file
+    width: 100%;
+    clear: both;
+    overflow-y: scroll;
+    height: 300px;
+}
```

Comment:
this works, but a more modern approach would be using flex box and/or grid 

https://css-tricks.com/snippets/css/complete-guide-grid/
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

not really necessary for such simple cases as this, but good to know